docs/man/man1/kubectl-expose.1

```diff
@@ -92,6 +92,10 @@ labels are specified, the new service will re\-use the labels from the resource
     Record current kubectl command in the resource annotation.
 
 .PP
+\fB\-R\fP, \fB\-\-recursive\fP=false
+    If true, process directory recursively.
```

Comment:
I modified expose to work with multiple resources, please see commits. Am I missing something?